### PR TITLE
WIP: Add OncoKB online updater

### PR DESCRIPTION
**Pull request by susannasiebert**
There are probable still some attributes etc we'd want to add but this is a first start.

### CODE REVIEW

File: lib/genome/online_updaters/oncokb/api_client.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    end
+
+    def drug_for_name(name)
+      get_data(drug_lookup_base_url, drug_params(name))
+    end
+
+    private
+    def get_data(base_url, params)
+      uri = URI.parse(base_url).tap do |u|
+        u.query = URI.encode_www_form(params)
```
Comment (ahwagner) [inline lib/genome/online_updaters/oncokb/api_client.rb]: `URI.encode_www_form` is new to me. This is super handy!

File: lib/genome/online_updaters/oncokb/updater.rb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  class Updater
+    attr_reader :new_version
+    def initialize(source_db_version = Date.today.strftime("%d-%B-%Y"))
+      @new_version = source_db_version
+    end
+
+    def update
+      remove_existing_source
+      source = create_new_source
+      create_interaction_claims(source)
```
Comment (ahwagner) [inline lib/genome/online_updaters/oncokb/updater.rb]: This is functionally valid. I'm curious why we're capturing and passing `source` here, though, when we have the `@source` instance variable defined at [Line 20](https://github.com/griffithlab/dgi-db/pull/233/files/1d3b361aad1a33dba44ed4a46b9f35de874bcece#diff-45d3567762a2cf334d0fdd1d1ee887d7R20)?

File: lib/genome/online_updaters/oncokb/updater.rb [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      api_client.variants.each do |variant|
+        gene = genes[variant['gene']]
+        gene_claim = create_gene_claim(gene, source)
+
+        variant['drugs'].split(', ').each do |drug_name|
+          if drug_name.include? '+'
+            combination_drug_name = drug_name
+            combination_drug_name.split(' + ').each do |individual_drug_name|
+              if valid_drug?(individual_drug_name)
+                puts(individual_drug_name)
```
Comment (ahwagner) [inline lib/genome/online_updaters/oncokb/updater.rb]: this was just for debug, right? we should consider removing it.

File: lib/genome/online_updaters/oncokb/updater.rb [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                  name: 'combination therapy',
+                  value: combination_drug_name,
+                  interaction_claim_id: interaction_claim.id,
+                ).first_or_create
+                add_interaction_claim_publications(interaction_claim, variant['pmids'])
+              end
+            end
+          else
+            if valid_drug?(drug_name)
+              puts(drug_name)
```
Comment (ahwagner) [inline lib/genome/online_updaters/oncokb/updater.rb]: another debug line?

**Comment by susannasiebert** [comment]
@ahwagner I fixed the merged conflict and addressed your comments.

**Pull request merged**